docker/nginx/default.conf

```diff
@@ -1,5 +1,5 @@
 server {
-    server_name tm.dev;
+    server_name tm.local;
```

Comment:
maybe would better to switch to self-signed SSL connection?

Reply:
@dominikhajduk ok, so please create a ticket with appropriate description

Reply:
not really, i've tested not signed certs and chrome still doesn't cooperate

Reply:
@myForksFiles co self signed do not work at all?